smoketests/__init__.py

```diff
@@ -23,16 +24,28 @@
 TEMPLATE_LIB_RS = open(STDB_DIR / "crates/cli/src/subcommands/project/rust/lib._rs").read()
 TEMPLATE_CARGO_TOML = open(STDB_DIR / "crates/cli/src/subcommands/project/rust/Cargo._toml").read()
 bindings_path = (STDB_DIR / "crates/bindings").absolute()
-escaped_bindings_path = re.escape(
-    str(bindings_path).replace('\\', '\\\\')
-)
+escaped_bindings_path = str(bindings_path).replace('\\', '\\\\\\\\') # double escape for re.sub + toml
```

Comment:
This was necessary because it turns out you're not supposed to use re.escape for the replacement end of a regex. It incorrectly escapes things like `-`; I ran into this because the path I was developing had a `-` in it.